modules/tinymce/src/core/main/ts/api/util/VK.ts

```diff
@@ -46,14 +48,16 @@ const VK: VK = {
   PAGE_DOWN: 34,
   END: 35,
   HOME: 36,
+  CTRL: 17,
+  META: 224,
 
-  modifierPressed: (e: KeyboardLikeEvent): boolean => {
+  modifierPressed: (e: (KeyboardLikeEvent & { keyCode?: number })): boolean => {
     return e.shiftKey || e.ctrlKey || e.altKey || VK.metaKeyPressed(e);
   },
 
-  metaKeyPressed: (e: KeyboardLikeEvent): boolean => {
+  metaKeyPressed: (e: (KeyboardLikeEvent & { keyCode?: number })): boolean => {
```

Comment:
This is a breaking change so not something we should be doing. This is used within the `modifierPressed` logic and a modifier isn't pressed if pressing the actual meta key.

To clarify, these aren't meant to be used to see if the key itself is pressed, it's meant to be used to see if the key is pressed in addition to another key.

Reply:
Any file that is in the api directory is a public api. So changing things there is more risky since third party might use that. Also agree with Lee here that it's supposed to check for modifiers states of key events so it behavior is being changed from that to check for modifiers and the modifier key it self. Just move the keyCode check to the normalize if statement where this is being called.

Reply:
Done!